include/junit_integration

```diff
@@ -61,6 +61,20 @@ prepare_junit_check_output() {
 
 finalise_junit_check_output() {
   echo '</testsuite>' >> "$JUNIT_OUTPUT_FILE"
+  TEST_COUNT="0"
+  TEST_SUCCESS_COUNT="0"
+  TEST_FAILURE_COUNT="0"
+  TEST_SKIPPED_COUNT="0"
+  TEST_ERROR_COUNT="0"
+  TEST_COUNT=$(grep -c "testcase name=" "$JUNIT_OUTPUT_FILE")
+  TEST_SUCCESS_COUNT=$(grep -A1 "testcase name=" "$JUNIT_OUTPUT_FILE" | grep -c "<system-out>")
+  TEST_FAILURE_COUNT=$(grep -A1 "testcase name=" "$JUNIT_OUTPUT_FILE" | grep -c "<failure message")
+  TEST_SKIPPED_COUNT=$(grep -A1 "testcase name=" "$JUNIT_OUTPUT_FILE" | grep -c "<skipped message")
+  TEST_ERROR_COUNT=$(grep -A1 "testcase name=" "$JUNIT_OUTPUT_FILE" | grep -c "<error message")
```

Comment:
Would it be better to maintain counters that increment on every pass/fail/skip? It's something that other formats could make use of, and avoids the need to use `grep` to count for XML tags that are somewhat side-effects of the pass/fail themselves (e.g. `<system-out>` for successes)

Reply:
Totally agree, have refined first cut with counters however this is limited to the junit output include.